web-react/src/components/DisplayChurchList.jsx

```diff
@@ -12,39 +13,111 @@ const DisplayChurchList = (props) => {
   }
 
   return (
-    <Container>
-      <h5 className="text-muted">{`${data[0].__typename} Locations:`}</h5>
-      <Row>
+    <Container className="mt-3">
+      <Row style={{ borderRadius: '3px' }}>
```

Comment:
Say no to inline styles. Amen

Reply:
And this even probably has an equivalent bootstrap class

Reply:
I thought I removed all